src/combinatorics/quiver/iso.lean

```diff
@@ -0,0 +1,186 @@
+/-
+Copyright (c) 2023 Adam Topaz, Rémi Bottinelli. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Adam Topaz, Rémi Bottinelli
+-/
+import combinatorics.quiver.basic
+import combinatorics.quiver.cast
+import logic.equiv.basic
+import tactic.nth_rewrite
+/-!
+# Isomorphisms of quivers
+
+Isomorphisms of quivers, defined as pairs of prefunctors that compose to the identities.
+
+## Main definitions
+
+* For quivers `U` and `V`, `quiver.iso U V` is the type of isomorphisms between `U` and `V`.
+* `quiver.iso.of_bijective` is the isomorphism defined by a prefunctor that is bijective on
+  vertices and arrows.
+
+## Notation
+
+* `U ≃q V` is notation for `quiver.iso U V`
+
+-/
+
+universes u v w z
+
+namespace quiver
+
+/--
+An isomorphism of quivers is given by a pair of prefunctors whose two compositions
+are the identities.
+-/
+structure iso (U V : Type*) [quiver.{u+1} U] [quiver.{v+1} V] extends prefunctor U V :=
```

Comment:
Is there any reason to restrict universes here? If I understand correctly, with these restrictions we can't add a projection `category_theory.equivalence.to_quiver_iso` that works for all universes.

Reply:
No, I think that's probably an oversight from my part.

Reply:
Splitting this `(U V : Type*)` seems to work, but I can't do the same below with `(U V W Z : Type*)` easily, so left that latter one as-is.